src/components/data-management/AnalysisModal.jsx

```diff
@@ -45,6 +49,81 @@ const NewExperimentModal = (props) => {
     rules.ALPHANUM_DASH_SPACE,
   ];
 
+  const renderAnalysisList = () => {
+    if (!experimentsList?.length > 0) {
```

Comment:
I would just check `=== 0` rather than negating a greater than

Reply:
Although, if `experimentsList` can be undefined, and it is, this condition would become false instead of true.

I would either remove the optional chain (if it in fact can't be undefined) or do `(experimentsList === undefined || experimentsList.length === 0)`

Reply:
Yes, this test is basically the shorter form of `(experimentsList === undefined || experimentsList.length === 0)`

Reply:
Oh right, didn't realise this was a deliberate JS hack!